docs/en/cookbook/entities-in-session.rst

```diff
@@ -29,40 +31,20 @@ the session into a managed Doctrine object looks like this:
     $em = GetEntityManager(); // creates an EntityManager 
 
     session_start();
-    if (isset($_SESSION['user']) && $_SESSION['user'] instanceof User) {
-        $user = $_SESSION['user']; 
-        $user = $em->merge($user);
+    if (isset($_SESSION['user'])) {
+        $user = $em->find(User::class, $_SESSION['user']);
```

Comment:
I wouldn't give `EntityManager#getReference()` to users unless they REALLY know what they are doing (hint: they don't)